README.md

```diff
@@ -1 +1,27 @@
+![PRIDEbot](https://raw.githubusercontent.com//pride-universe/PRIDEbot/master/PRIDEbot.jpg)
+
 # PRIDEbot
+A discord bot for managing the [Enbies R Us discord server](https://discord.gg/Q4gabhP).
```

Comment:
It's actually for a whole network of servers, collectively called "Pride Universe" (thus the name of the bot PRIDEbot")
```
Dudes R Us - A trans-masc focused server
https://discord.gg/SGKujXY

Gals R Us - A trans-femme focused server
https://discord.gg/BEQGMQh

Enbies R Us - A nonbinary focused server
https://discord.gg/HKQgkn8

Trans + Allies - An education focused server for partners, parents, and allies
https://discord.gg/dqyuc7S
```